projects/kserve/testing/test_e2e.py

```diff
@@ -151,12 +151,22 @@ def deploy_and_test_models_e2e():
     logging.info("Wait 60s for Prometheus to restart collecting data ...")
     time.sleep(60)
 
+    using_longevity = config.ci_artifacts.get_config("tests.e2e.longevity.enabled")
+
     try:
         deploy_models_concurrently()
 
-        test_models_concurrently()
+        if not using_longevity:
+            test_models_concurrently()
+            test_models_sequentially(locally=False)
+        else:
+            repeats = config.ci_artifacts.get_config("tests.e2e.longevity.repeats")
+            delay = config.ci_artifacts.get_config("tests.e2e.longevity.delay")
+            for i in range(repeats):
+                test_models_concurrently()
+                time.sleep(delay)
```

Comment:
can you add:
```suggestion
                if i != repeats - 1:
                    time.sleep(delay)
```
(if I'm not mistaking, this should avoid a useless delay)

Reply:
maybe you can move these lines of code into a dedicated function, and rewrite it the other way:
```
if using_longevity:
        test_models_longevity()
else:
        test_models_concurrently()
        test_models_sequentially(locally=False)
```
(I try to avoid negative conditions, whenever possible)